include/avif/internal.h

```diff
@@ -732,7 +732,7 @@ avifResult avifRWStreamWriteU32(avifRWStream * stream, uint32_t v);
 avifResult avifRWStreamWriteU64(avifRWStream * stream, uint64_t v);
 avifResult avifRWStreamWriteZeros(avifRWStream * stream, size_t byteCount);
 // The following functions can write non-aligned bits.
-avifResult avifRWStreamWriteBits(avifRWStream * stream, uint32_t v, size_t bitCount);
+AVIF_NODISCARD avifResult avifRWStreamWriteBits(avifRWStream * stream, uint32_t v, size_t bitCount);
```

Comment:
I thought there was a compiler restriction I had missed because there are other `AVIF_NODISCARD avifResult` in the file but no. Indeed, `avifResult` is enough.